### ISSUE TITLE
Audit component library in high-contrast/forced colors mode

### ISSUE THREAD

**Issue by amyleadem** [department-of-veterans-affairs/vets-design-system-documentation#4783]
## Description
Audit the design system component library in forced colors mode.

> "Forced colors mode is an accessibility feature intended to increase the readability of text through color contrast. Individuals with limited vision often find it more comfortable to read content when there is a particular type of contrast between foreground and background colors." [^1]

Note that at this time there is no current WCAG criterion that explicitly states a need for forced color mode support, but WebAIM identified in a 2018 survey that [30% of low-vision users use forced colors mode](https://webaim.org/projects/lowvisionsurvey2/#at), and one source cited that [Microsoft reported that 4% of users use high-contrast mode](https://cssence.com/2024/forced-colors-mode-strategies/). 

[^1]: https://www.w3.org/TR/css-color-adjust-1/#forced

## Tasks

- [ ] Check all design system components and variants in forced colors mode
    - Windows offers a handful of [contrast themes as a system setting](https://support.microsoft.com/en-us/windows/change-color-contrast-in-windows-fedc744c-90ac-69df-aed5-c8a90125e696). If you are not on a Windows machine, you can [emulate forced colors mode via Chrome DevTools](https://developer.chrome.com/docs/devtools/rendering/emulate-css). 
- [ ] Create a report that identifies:
    - [ ] Accessibility failures, where a critical/meaningful element is no longer visible
    - [ ] Accessibility enhancements, where designating system colors might improve understanding for HCM users

## Acceptance Criteria
- [ ] We have a report that catalogs all accessibility failures related to forced colors mode
- [ ] We have opened issues to remediate any failures and enhancements
